client-static/src/test/kotlin/app/cash/backfila/client/static/StaticKotlinValBackfillTest.kt

```diff
@@ -97,11 +97,13 @@ class StaticKotlinValBackfillTest {
 
       this.assertThatCode {
         backfila.createWetRun<SaucesBackfill>(rangeStart = "abc")
-      }.hasMessageContaining("must be a number")
+      }.isInstanceOf(NumberFormatException::class.java)
+        .hasMessageContaining("For input string")
```

Comment:
Re my other comment, I think we can retain the information about start/end needing to be a number, rather than throwing some generic "couldn't parse input string" message